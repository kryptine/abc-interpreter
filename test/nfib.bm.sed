s/30/43/
