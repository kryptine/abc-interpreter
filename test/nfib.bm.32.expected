1402817465
halt at 6
3 524285 524288
